freelens-launch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+EXTRA_ARGS+=(
+  "--no-sandbox"
```

Comment:
use zypak it's in the electron baseapp, drop no-sandbox.

Reply:
Yes, all is fine without --no-sandbox. Thanks for suggestion!